roles/horizon/templates/etc/openstack-dashboard/local_settings.py

```diff
@@ -180,6 +180,25 @@
 # of your entire OpenStack installation, and hopefully be in UTC.
 TIME_ZONE = "UTC"
 
+#
+# The Horizon Policy Enforcement engine uses these values to load per service
+# policy rule files. The content of these files should match the files the
+# OpenStack services are using to determine role based access control in the
+# target installation.
+
+# Path to directory containing policy.json files
+POLICY_FILES_PATH = "/opt/stack/horizon/conf"
+# Map of local copy of service policy files
+POLICY_FILES = {
+    'identity': 'keystone_policy.json',
+    'compute': 'nova_policy.json',
+    'volume': 'cinder_policy.json',
+    'image': 'glance_policy.json',
+    'orchestration': 'heat_policy.json',
+    'network': 'neutron_policy.json',
+    'telemetry': 'ceilometer_policy.json',
+}
```

Comment:
The paths here don't seem to match the symlinks created in the previous task ( https://github.com/blueboxgroup/ursula/pull/1222/files#diff-09f0a1ed1ee264fcf97572911b2f3b2dR130 ). Why is there a difference?

Also, if you can configure Horizon to look in a specific place for the policy files, why delete the ones it ships? Why not just install / link the new policy files and then point to them in this config?